packaging/installer/netdata-updater.sh

```diff
@@ -769,8 +769,8 @@ update_binpkg() {
     opensuse)
       pm_cmd="zypper"
       repo_subcmd="--gpg-auto-import-keys refresh"
-      upgrade_cmd="upgrade"
-      pkg_install_opts="${interactive_opts} --allow-unsigned-rpm"
+      upgrade_cmd="update"
+      pkg_install_opts="${interactive_opts}"
```

Comment:
Until we get #14256 fixed, we probably need to keep the `--allow-unsigned-rpm` option here.

Reply:
I think it worked fine when I tested it but I can replace `update` with `install` then, since `update` doesn't have a `--allow-unsigned-rpm` option? Do you see any problem with that?

Reply:
Ah, yes, you’re correct, we do not want that option here.